src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs

```diff
@@ -21,75 +22,88 @@ public CentOpsFixture(IConfiguration configuration, TestClients testClients)
-                Status = "Active",
-                ApiKey = "thisisareallylongkey"
-            });
-            _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, dmrPostBody).Result;
-
-            // Create Bot1
-            var bot1PostBody = JsonSerializer.Serialize(new Participant()
-            {
-                Name = "bot1",
-                InstitutionId = institution.Id,
-                Host = "http://bot1/dmr-api/messages",
-                Type = "Chatbot",
-                Status = "Active",
-                ApiKey = "thisisareallylongkeyformockbot1"
-            });
-            _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, bot1PostBody).Result;
+
+                // Create Classifier
+                var classifierPostBody = JsonSerializer.Serialize(new Participant()
+                {
+                    Name = $"classifier1",
+                    InstitutionId = institution.Id,
+                    Host = $"{_configuration["ClassifierInternalUrl"]}/dmr-api/messages",
+                    Type = "Classifier",
+                    Status = "Active",
+                    ApiKey = "thisisareallylongkeyforclassifier"
+                });
+                _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, classifierPostBody).Result;
+
+                // Create Dmr
+                var dmrPostBody = JsonSerializer.Serialize(new Participant()
+                {
+                    Name = $"dmr1",
+                    InstitutionId = institution.Id,
+                    Host = $"{_configuration["DmrInternalUrl"]}/messages",
+                    Type = "Dmr",
+                    Status = "Active",
+                    ApiKey = "thisisareallylongkey"
+                });
+                _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, dmrPostBody).Result;
+
+                // Create Bot1
+                var bot1PostBody = JsonSerializer.Serialize(new Participant()
+                {
+                    Name = $"bot1",
+                    InstitutionId = institution.Id,
+                    Host = $"{_configuration["Bot1InternalUrl"]}/dmr-api/messages",
+                    Type = "Chatbot",
+                    Status = "Active",
+                    ApiKey = "thisisareallylongkeyformockbot1"
+                });
+                _ = _testClients.CentOpsAdminClient.Request<Participant>(Verb.Post, _participantsUri, bot1PostBody).Result;
+            }
         }
 
         public void Dispose()
         {
             // Do "global" teardown here; Only called once.
 
-            // Get all participant for test id
-            var participants = _testClients.CentOpsAdminClient.Request<List<Participant>>(Verb.Get, _participantsUri).Result;
-
-            // Delete each participant
-            foreach (var participant in participants)
+            if (_useFixture)
```

Comment:
Might make it easier to read if we invert this case and return early.  Same with the other usage above - up to you entirely.

Reply:
Done for the constructor. The Dispose method only uses the `true` path so nothing to revert.